libmscore/instrument.cpp

```diff
@@ -25,8 +25,8 @@
 
 namespace Ms {
 
-const char* Channel::DEFAULT_NAME = "normal";
-const char* Channel::HARMONY_NAME = "harmony";
+const char* Channel::DEFAULT_NAME = QT_TRANSLATE_NOOP("channel", "Normal");
+const char* Channel::HARMONY_NAME = QT_TRANSLATE_NOOP("channel", "Chord symbols");
```

Comment:
a) Sentence case
b) Translatable
c) Using the 'official' UI term rather then the internal code name

Reply:
I think it’s not a good idea to use the name for the user instead of identifiers. It may be better to separate the identifiers and the name for users, i.e. add a method  

```
QString channelTitle(const char* name) const 
{
    if (Channel::DEFAULT_NAME == name)    return QT_TRANSLATE_NOOP("channel", "Normal");
    if (Channel::HARMONY_NAME == name) return QT_TRANSLATE_NOOP("channel", "Chord symbols");
    return name;
}

Reply:
Only that translations don't work that way I'm affraid

Reply:
even easier:  
```
QString channelTitle(const char* name) const 
{
    if (Channel::DEFAULT_NAME == name) return qApp->translate("channel", "Normal");
    if (Channel::HARMONY_NAME == name) return qApp->translate("channel", "Chord symbols");
    return name;
}
```

Reply:
> Check how it is done for other such strings.
> At the very least here it's be `tr("Normal")`, but then all the other places need to change too

I saw, OK, rejecting my suggestion

Reply:
> even easier:
Not easier but wronger ;-)